vwifi.c

```diff
@@ -132,14 +175,32 @@ static void owl_scan_routine(struct work_struct *w)
  * This routine is called through workqueue, when the kernel asks to connect
  * through cfg80211_ops.
  */
+static bool is_valid_ssid(const char *connecting_ssid)
+{
+    bool is_valid = false;
+
+    size_t len = strlen(ssid_list);
+    char *sdssid_copy = kmalloc((len + 1) * sizeof(sdssid_copy), GFP_KERNEL);
+    strncpy(sdssid_copy, ssid_list, len);
+    for (char *sdssid = strsep(&sdssid_copy, sdssid_delim); sdssid != NULL;
```

Comment:
Change from `sdssid != NULL` to `sdssid`.

Reply:
fixed on https://github.com/sysprog21/vwifi/pull/4/commits/79864489e79627b76f1628ebbdeda20ca1d1b985